content/en/tracing/profiling/_index.md

```diff
@@ -129,6 +129,59 @@ The Datadog Profiler requires Python 2.7+. Memory profiling only works on Python
 [3]: /tracing/visualization/#services
 {{% /tab %}}
 
+{{% tab "Go" %}}
+
+The Datadog Profiler requires Go 1.12+. To begin profiling applications:
+
+1. Install `ddtrace` which contains both tracing and profiling:
+
+    ```shell
+    go get gopkg.in/DataDog/dd-trace-go.v1/ddtrace
+    ```
+
+     **Note**: Profiling is available in the `ddtrace` library for versions 1.23.0+.
```

Comment:
Again, it's not called "`ddtrace` library". We can maybe also say:
```suggestion
     **Note**: Profiling is available in `dd-trace-go` starting version v1.23.0.
```

Reply:
+1. Fwiw we had feedback from docs folks on an earlier pr that they liked formatting like `**Note**: Profiling is available in `dd-trace-go` v1.23.0+.`

Reply:
Done